src/components/description/MbtiDetail.tsx

```diff
@@ -9,13 +9,13 @@ import Header from "../common/Header";
 const MbtiDetail = () => {
   const { type } = useParams();
```

Comment:
이렇게 하면 `type`에 대한 type이 정의 될거에요 
```suggestion
  const { type } = useParams<{type?: keyof typeof validMbti}>();
```

validMbti를 가져올때 타입을 미리 만들어서 import하는 것이 좋습니다
```typescript
const validMbti = {
...
} as const;

type ValidMbti = keyof typeof validMbti;
```

Reply:
const { type } = useParams<{ type?: string }>();
  const mbtiType = type?.toUpperCase() as keyof typeof validMbti;
  const description = validMbti[type?.toUpperCase() as keyof typeof validMbti];
  
  이렇게 가져올 때 타입을 추가하였습니다!